### PR TITLE
Adding wofdata support for op-build packages

### PR DESCRIPTION
**Pull request by e-liner**
This commit makes all necessary changes for op-build (makefile, configuration names and the like) and additionally brings in changes from the pnor repository.  Ultimately, this patch will make it possible to fill in the WOFDATA section of the pnor.

---
This change is [Reviewable](https://reviewable.io/reviews/open-power/op-build/1094)

### CODE REVIEW

**Comment by e-liner** [comment]
retest this please

**Comment by e-liner** [comment]
retest this please

**Comment by e-liner** [comment]
I also tested this out locally by adding the csv file to the machine-xml package, witherspoon specifically, and op-build built the WOFDATA partition and added it to the pnor.

File: openpower/package/machine-xml/machine-xml.mk [hunk 1]
```diff
@@ -91,6 +91,16 @@ define MACHINE_XML_BUILD_CMDS
             $(PETITBOOT_BIOS_XML_METADATA_FILE) \
             $(PETITBOOT_XSLT_FILE) \
             $(BIOS_XML_METADATA_FILE)
+
+        # Create the wofdata
+        if [ -e $(MRW_HB_TOOLS)/wof-tables-img ]; then \
+            chmod +x $(MRW_HB_TOOLS)/wof-tables-img; \
+        fi
+
+        if [ -e $(MRW_SCRATCH)/wof_v4_dd1_export.csv ]; then \
```
Comment (dcrowell77) [inline openpower/package/machine-xml/machine-xml.mk]: The filename of the csv should be more generic, this name has a version and dd level in there.  The revision will change a lot.  I think we have 2 options:
a) use a common generic name for everything - wofdata.csv?
b) create a variable to use a machine-specific filename, e.g. wofdata_witherspoon.csv

Comment (e-liner) [inline openpower/package/machine-xml/machine-xml.mk]: I like the generic name. Especially since everything is going through machine.xml anyway.

**Comment by e-liner** [comment]
retest this please

**Pull request merged**